README.md

```diff
@@ -27,7 +27,7 @@ CentOS Linux 8. It does not support CentOS Stream.
 ## Usage
 
 1. Login to your CentOS Linux 6, 7 or 8 instance as a user who has `sudo` privileges.
-1. Either clone this repository or download the [`centos2ol.sh`](./centos2ol.sh) script.
+1. Either clone this repository or download the [`centos2ol.sh`][1] script.
```

Comment:
Why not just replace this hyperlink instead of having a reference? [filename](fullrawpath)

Reply:
GitHub doesn't have a magic way of creating raw links (AFAIK) and the link is very long, so I figured it would be better not to annoy `markdownlint` with a very, very long line.

Reply:
But can't we just use
 `[`centos2ol.sh`](https://raw.githubusercontent.com/oracle/centos2ol/main/centos2ol.sh)`

Reply:
It was explained to me that both methods render the same, I mistakenly believed the use of [references] to be an appendix